examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+is a Pachyderm technique for controlling when data gets processed.
+Transactions are Pachyderm feature for assuring data from different repos get processed together.
+
+This is an example which uses a simple DAG, 
+based on our opencv example, 
+to illustrate using deferred processing along with transactions.
+
+
+
+## Pipelines
+
+![Example DAG](./example_dag.png)
+
+The DAG shown is a simple elaboration on the opencv example,
+with pipeline and repo names chosen to avoid collisions with that example 
+if installed in the same cluster.
+
+The functionality is, of course, different.
+The `edges_dp` pipeline will do edge detection on images committed to `images_dp_1`.
+`montage_dp` will create a montage out of images committed to `images_dp_2` and images in the master branch of `edges_dp`. 
+This is so it's easy to verify the files being processed
+by visually inspecting the montage.
+
+The most significant change from opencv in this example is the pipeline spec for `edges_dp`,
+which has the `output_branch` attribute set to `dev`.
+
+```json hl_lines="11"
+{
+    "pipeline": {
+        "name": "edges_dp"
+    },
+    "input": {
+        "pfs": {
+            "glob": "/*",
+            "repo": "images_dp_1"
+        }
+    },
+    "output_branch": "dev",
+    "transform": {
+        "cmd": [ "python3", "/edges.py" ],
+        "image": "pachyderm/opencv"
+    }
+}
+```
+
+This means that this pipeline, 
+instead of putting its output in the `master` branch,
+will put it in a branch called `dev`.
+
+Since `montage_dp` is subscribed to the master branch of `edges_dp`,
+we can simply create a `master` branch attached to any commit in `edges_dp` to trigger the pipeline.
+
+## Example run-through
+
+### Deferred Processing
+
+You should have a Pachyderm cluster set up 
+and access to it configured from your local computer
+prior to running this example.
+
+1. Run the script `setup.sh` included in this repo.
+   It will perform the following steps for you:
+   
+   ```sh
+   pachctl create repo images_dp_1
+   pachctl create repo images_dp_2
+   pachctl create pipeline -f ./edges_dp.json 
+   pachctl create pipeline -f ./montage_dp.json 
+   pachctl put file images_dp_1@master -i ./images.txt
+   pachctl put file images_dp_1@master -i ./images2.txt
+   pachctl put file images_dp_2@master -i ./images3.txt
+   pachctl create branch edges_dp@stable_1_0 --head dev^
+   ```
+   
+2. Once the demo is loaded, 
+   look at the commits in `edges_dp`.
+   They should look something like this.
+   
+   ```sh
+   $  pachctl list commit edges_dp
+   REPO         BRANCH COMMIT                           FINISHED      SIZE PROGRESS DESCRIPTION
+   edges_dp     dev    85ca689439b943978d273b311003d1bf 2 minutes ago 0B   -         
+   edges_dp     dev    dc23aa4da6b94fefbebb6751c1c9b5a3 2 minutes ago 0B   -         
+   ```
+   
+3. Remember that the `edges_dp` pipeline outputs to the `dev` branch.
```

Comment:
This is not a step but an explanatory text, so remove the number but keep the indentation.